aws-opentelemetry-distro/src/amazon/opentelemetry/distro/aws_opentelemetry_distro.py

```diff
@@ -36,6 +40,21 @@ def _configure(self, **kwargs):
             OTEL_EXPORTER_OTLP_METRICS_DEFAULT_HISTOGRAM_AGGREGATION environment variable. Need to work with upstream to
             make it to be configurable.
         """
+
+        # related issue: https://github.com/open-telemetry/opentelemetry-python-contrib/issues/2495
+        # mimicking what is done here: https://tinyurl.com/54mvzmte
+        # For handling applications like django running in containers, we are setting the current working directory
+        # to the sys.path for the django application to find its executables.
+        #
+        # Note that we are updating the sys.path and not the PYTHONPATH env var, because once sys.path is
+        # loaded upon process start, it doesn't refresh from the PYTHONPATH value.
+        #
+        # To be removed once the change has been contributed to upstream.
```

Comment:
Did we raise an upstream PR? If so can we link it here?

Also lets avoid using the term upstream, it's vague. Lets specify the repo or class that needs this change.

Reply:
I did't raise a PR in OTel yet. Will do so soon and link it here.